arm-recoveryservicesbackup/2016-06-01/swagger/recoveryservicesbackup.json

```diff
@@ -0,0 +1,1548 @@
+    "x-ms-code-generation-settings": {
+      "internalConstructors": true
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json",
+    "application/octet-stream"
+  ],
+  "produces": [
+    "application/json",
+    "text/json",
+    "application/octet-stream"
+  ],
+  "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceNamespace}/vaults/{resourceName}/backupFabrics/{fabricName}/protectionContainers/{containerName}": {
+      "get": {
+        "tags": [
+          "ProtectionContainers"
+        ],
+        "description": "Gets details of the specific container registered to your Recovery Services Vault.",
+        "operationId": "ProtectionContainers_BMSGetContainer",
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "name": "resourceName",
+            "in": "path",
+            "description": "The name of the recovery services vault.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ResourceNamespace"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of the resource group where the recovery services vault is present.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "name": "fabricName",
+            "in": "path",
+            "description": "Name of the fabric where the container belongs",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "containerName",
+            "in": "path",
+            "description": "Name of the container whose details need to be fetched",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Resource[ProtectionContainer]"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceNamespace}/vaults/{resourceName}/backupFabrics/{fabricName}/protectionContainers/{containerName}/protectedItems/{protectedItemName}": {
+      "get": {
+        "tags": [
+          "ProtectedItems"
+        ],
+        "description": "Provides the details of the backed up item. This is an asynchronous operation. To know the status of the operation, call the GetItemOperationResult API.",
+        "operationId": "ProtectedItems_GetProtectedItem",
```

Comment:
In AutoRest, we split the operation id on the underscore character and expect that it will follow the pattern "Noun_Verb". We generally recommend avoiding duplicating the noun in the verb part too: https://github.com/Azure/autorest/wiki/operation-id-noun-in-verb